build_dirs/build_docs.py

```diff
@@ -0,0 +1,50 @@
+import os
+import shutil
+import tarfile
+import json
+
+BASE = os.path.abspath(os.path.dirname(__file__))
```

Comment:
@macartur let's try to prioritize the use of PathLike objects for dealing with paths/dirs/files.

Can you please rewrite this PR considering this issue? Thanks! =)